integration/integration_test.go

```diff
@@ -0,0 +1,145 @@
+package integration
+
+import (
+	"encoding/hex"
+	"encoding/json"
+	"log"
+	"testing"
+	"time"
+
+	"github.com/CityOfZion/neo-go/pkg/core"
+	"github.com/CityOfZion/neo-go/pkg/core/transaction"
+	"github.com/CityOfZion/neo-go/pkg/io"
+	"github.com/docker/go-connections/nat"
+	"github.com/stretchr/testify/require"
+)
+
+// Main steps for testing are:
+// - prepare docker image with neo-go node or c# node
+// - start privatenet
+// - start docker nodeContainerReq
+// - create RPC client
+// - generate input for RPC client
+// - start sending txes to the node
+// - measure how much TX could be sent
+
+const (
+	// RpcPort for RPC calls to test nodes.
+	RpcPort = nat.Port("20331")
+	// NumberOfTx number of transactions to be generated.
+	NumberOfTx = 3000
```

Comment:
Just 3000?

Reply:
true, so my guess is that anyone will use this test with changing this number.
Like run with 3000 TX after with 3000+N and after with 3000+2*N and get some average number.